tools/prove_wrapper

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# shellcheck disable=SC2048,SC2086
+
+echo "Running prove with TAP output check"
+OUTPUT=$(prove "$@" 2>&1)
```

Comment:
This seems bad as we would lose the ability to follow the output as it is written.